docs/pages/2019_MARVEL_Psik_MaX/sections/appendix_restarting_calculations.rst

```diff
@@ -84,4 +84,5 @@ Finally, to submit the restart, since it is a process builder, it works exactly
 
 You can now inspect the restarted calculation to verify that this time it actually completed successfully.
 Using the restart builder, the required code to setup a calculation is much shorter than the one needed to launch a new one from scratch.
-There is no need to load and or create many of the inputs such as the pseudopotentials, structures and k-points, which were reused from the first calculation
+There is no need to load or create many of the inputs such as the pseudopotentials, structures and k-points,
```

Comment:
All good, but please keep each sentence on a single line.

Reply:
What happened to line length limits in code?

Reply:
I think both approaches are fine - we're just talking about the source of the tutorial here guys

Reply:
Line length limits for 'code' are good. This is text and there is a soft convention there to keep a single sentence per line, which works well with version control. As Leo said, it is just the source, just thought I would mention it

Reply:
Ok, good to know. Thanks,